wasm/tests/test.c

```diff
@@ -1695,6 +1695,11 @@ void test_object(void)
     opa_object_insert(obj, opa_string_terminated("b"), opa_number_int(2));
     opa_object_insert(obj, opa_string_terminated("c"), opa_number_int(3));
 
+    test("object/get (key found)", opa_value_compare(opa_obj_get(&obj->hdr, opa_string_terminated("a"), opa_number_int(2)), opa_number_int(1)) == 0);
+    test("object/get (string key not found)", opa_value_compare(opa_obj_get(&obj->hdr, opa_string_terminated("d"), opa_number_int(2)), opa_number_int(2)) == 0);
+    test("object/get (integer key not found)", opa_value_compare(opa_obj_get(&obj->hdr, opa_number_int(1), opa_number_int(2)), opa_number_int(2)) == 0);
+    test("object/get (boolean default value)", opa_value_compare(opa_obj_get(&obj->hdr, opa_number_int(1), opa_boolean(TRUE)), opa_boolean(TRUE)) == 0);
```

Comment:
❔ Worth adding a case for the non-object operand 1? What do you think?

Reply:
+1 from a coverage POV, it would be worth just adding that case.